README.md

```diff
@@ -1,7 +1,7 @@
 Project Ramda
 =============
 
-A practical functional library for Javascript programmers.
+Ramda is the simpler & more practical functional library for Javascript.
```

Comment:
to my mind, "functional" trumps "practical" every time. but i also think "practical" is even less well-defined than "functional". But hell, it's just a slogan. Maybe: "quaeritur de nostra Latina slogan certamen"